doc/source/barcelona.md

```diff
@@ -0,0 +1,58 @@
+
+![Barcelona architecture](_static/images/barcelona/arct.jpg)
+
+The ancient walls were built in the 12th and 13th centuries to defend the populace from attacks by foreign armies. Other famous buildings include the most extensive library in Europe, Casey Mila, Barcelona’s Cathedral, and Park Güell – one of Gaudí’s masterpieces, called an architectonic masterpiece.
+
+## FC Barcelona (Barça)
+
+FC Barcelona is a Spanish football club. The club enjoys massive support throughout the world, having won many trophies. FC Barcelona has had many top players, including Pep Guardiola and Lionel Messi.
+
+![Leo Messi](_static/images/barcelona/messi.jpg)
+
+FC Barcelona certainly has its unique characteristics, as demonstrated by its successful team and the individuals who have played for it. In fact, FC Barcelona is one of the most valuable sports franchises in Europe. Additionally, the Camp Nou stadium and museum rank among Barcelona’s top 10 tourist attractions.
+
+## Location and Climate
+
+The city of Barcelona sits on the Mediterranean coast, surrounded by mountains that form the province of Catalonia. 
+
+![Mediterranean coast](_static/images/barcelona/climate.jpg)
+
+The Mediterranean’s climate, with warm sunny days and cool nights in winter, explains the abundant greenery throughout most of the year. The climate is cooler at night from December to March, but even then, it rarely snows.
+Because of the city’s moderate temperature, residents spend a lot of time outside, from dining al fresco at outdoor restaurants and pubs to hiking or biking in the neighboring mountains. 
+
+## Festivals and Street Parties
+
+Apart from sports and architecture, the city of Barcelona is renowned for being one of the most energetic and active cities globally. The city has a range of festivals and carnival-style street festivities throughout the year, derived from traditions. 
+The city hosts several significant music events throughout the summer, including Primavera Sound, Sonar Festival, Cruella, and Rock Fest Barcelona. Festival lineups feature the biggest stars in the music business and draw up to 200,000 people to the city.
+
+![Barcelona festivals](_static/images/barcelona/festival.jpg) 
+
+Along with the music festivals, each city’s seventy-two wards has a Festa Major. These neighborhood fiestas feature street parties, parades, competitions for human towers, fire races, and fireworks displays.
+
+## Mediterranean Beaches 
+
+The city of Barcelona is renowned worldwide for its magnificent coastline. The city has a long, wild beach known as La Barceloneta. The Barceloneta is a highly famous beach in Barcelona, and it becomes quite crowded on weekends because of its easily accessible position. People come here to sunbathe and drink at one of the beachside bars and cafés or have a feast at one of the seafood restaurants in the neighboring tiny alleys. 
```

Comment:
```suggestion
The city of Barcelona is renowned worldwide for its magnificent coastline. The city has a long, wild beach known as La Barceloneta. The Barceloneta is a highly famous beach in Barcelona, and it becomes quite crowded on weekends because of its easily accessible position. People come here to sunbathe and drink at one of the beachside bars and cafés or have a feast at one of the seafood restaurants in the neighboring tiny alleys.
```